frontend-student/src/components/courseDetails/InstructorDetails.jsx

```
@@ -0,0 +1,45 @@
+      <div className='p-6 bg-blue-600 text-white'>
+        <h3 className='text-xl font-bold'>Meet Your Instructor</h3>
+      </div>
+      <div className='p-6'>
+        <div className='flex items-start'>
+          <div className='bg-blue-100 rounded-full p-3 mr-4'>
+            <svg
+              className='w-8 h-8 text-blue-600'
+              fill='none'
+              stroke='currentColor'
+              viewBox='0 0 24 24'
+            >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)